### PR TITLE
View everything

### PR DESCRIPTION
**Pull request by PguOrange**

### CODE REVIEW

File: app/src/main/java/com/example/newsandroid/database/DBProvider.kt [hunk 1]
```diff
@@ -6,7 +6,9 @@ import androidx.room.Room
 class DBProvider {
     companion object{
         private lateinit var INSTANCE: NewsDatabase
+        private lateinit var INSTANCE_EVERY: EverythingNewsDatabase
         private val dbName = "news"
+        private val dbNameEverything = "newsEverything"
```
Comment (PguOrange) [inline app/src/main/java/com/example/newsandroid/database/DBProvider.kt]: Pourquoi 2 bases de données ?

File: app/src/main/java/com/example/newsandroid/repository/NewsRepository.kt [hunk 2]
```diff
@@ -42,4 +46,20 @@ class NewsRepository(private val database: NewsDatabase, private val newsApiServ
         return size
     }
 
+    suspend fun refreshNewsEverything() {
+        withContext(Dispatchers.IO) {
+            try {
+                val newsCollection = newsApiService?.getEverything("bitcoin")?.await()
```
Comment (PguOrange) [inline app/src/main/java/com/example/newsandroid/repository/NewsRepository.kt:47]: Pourquoi `bitcoin` en tant que query ?

File: app/src/main/java/com/example/newsandroid/database/DBProvider.kt (hunk 1 above)
Comment (martinsbenoit) [reply app/src/main/java/com/example/newsandroid/database/DBProvider.kt]: 606f2cf77cc16a4b65974de557ba3f3d7b02a41a

**Comment by PguOrange** [comment]
[Vue EveryThing](https://trello.com/c/KYW3KgAT/9-vue-everything)

**Review by PguOrange — COMMENTED** [review]
OK

**Comment by PguOrange** [comment]
[Vue EveryThing](https://trello.com/c/KYW3KgAT/9-vue-everything)

**Pull request merged**